File: javascript/problems/arrays/reverse.js
function reverse(arr = [], count) {
    if (count === len) return arr

    let current = arr[count]

    count++
    reverse(arr, count)
    arr[len - 1 - (count - 1)] = current
    return arr
}

let arr = [22, 11, 20, 76, 123, 70]
let len = (arr && arr.length) || 0

let reversedArray = reverse(arr, (count = 0))
console.log('Reversed Array => ', reversedArray)

File: python/problems/array/reverse.py
def reversed_arr(arr,count):
	if count==len(arr):
		return arr
	temp = arr[count]
	count += 1
	reversed_arr(arr,count)
	arr[len(arr) - 1 - (count - 1)] = temp
	return arr

arr = [22, 11, 20, 76, 123, 70]
count=0
print ("Reversed array: ",reversed_arr(arr,count))